packages/upload-api/src/types.ts

```diff
@@ -515,7 +515,12 @@ export interface StoreAddInput {
   link: UnknownLink
   size: number
   origin?: UnknownLink
-  issuer: DID
+
+  /**
+   * @deprecated - Issuer of the invocation is irrelevant as long as
+   * they have authorization to invoke on subject `space`.
+   */
+  issuer?: DID
```

Comment:
Ideally we'd like tracability with a `cause` field but probably not so bad now we're moving to blob protocol.

Reply:
that would be the invocation right?

Reply:
Oh, yeah I'm getting mixed up with the upload table. Ignore me!